projects/commudle-admin/src/app/feature-modules/sys-admin/components/admin-badges/admin-badges-assign/admin-badges-assign.component.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, OnInit, TemplateRef, ViewChild } from '@angular/core';
+import { IUserBadge } from 'projects/shared-models/user_badge.model';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { LibToastLogService } from 'projects/shared-services/lib-toastlog.service';
+import { Subscription } from 'rxjs';
+import { NbDialogService } from '@nebular/theme';
+import { SysAdminUserBadgesService } from 'projects/commudle-admin/src/app/feature-modules/sys-admin/services/sys-admin-user-badges.service';
+
+@Component({
+  selector: 'app-admin-badges-assign',
+  templateUrl: './admin-badges-assign.component.html',
+  styleUrls: ['./admin-badges-assign.component.scss']
+})
+export class AdminBadgesAssignComponent implements OnInit {
+
+  user_badges: IUserBadge[] = [];
```

Comment:
Why have we named this differently than the other variables?

Reply:
converted to camel case